plugins/spu2-x/src/Windows/resource.h

```diff
@@ -47,8 +46,6 @@
 #define IDC_MSG_PUBLIC_BUILD            1048
 #define IDC_XA2_TRIBLE_BUFFER           1050
 #define IDC_LABEL_VERSION_INFO          1054
```

Comment:
This should also be removed too.

Reply:
OK, what about IDC_COMBO1? It's present in Lilypad and USBQemu, but it isn't used for anything at least in SPU2-X's case. Should I remove that as well?

Reply:
It's up to you.

Reply:
If it were me I'd remove it, no need to keep useless code.